client/lib/upgrades/constants.js

```diff
@@ -16,40 +25,26 @@ module.exports.action = keyMirror( {
 	EMAIL_FORWARDING_FETCH: null,
 	EMAIL_FORWARDING_FETCH_COMPLETED: null,
 	EMAIL_FORWARDING_FETCH_FAILED: null,
-	DOMAIN_ENABLE_LOCKING_COMPLETED: null,
-	DOMAIN_ENABLE_PRIVACY_PROTECTION_COMPLETED: null,
-	DOMAIN_TRANSFER_ACCEPT_COMPLETED: null,
-	DOMAIN_TRANSFER_CODE_REQUEST_COMPLETED: null,
-	DOMAIN_TRANSFER_DECLINE_COMPLETED: null,
-	FETCH_DNS: null,
-	FETCH_GOOGLE_APPS_USERS: null,
+	GOOGLE_APPS_USERS_FETCH: null,
+	GOOGLE_APPS_USERS_FETCH_COMPLETED: null,
+	ICANN_VERIFICATION_RESEND_COMPLETED: null,
 	NAMESERVERS_FETCH: null,
 	NAMESERVERS_FETCH_COMPLETED: null,
 	NAMESERVERS_FETCH_FAILED: null,
 	NAMESERVERS_UPDATE_COMPLETED: null,
-	OPEN_CART_POPUP: null,
 	PRIMARY_DOMAIN_SET: null,
 	PRIMARY_DOMAIN_SET_COMPLETED: null,
 	PRIMARY_DOMAIN_SET_FAILED: null,
-	PURCHASES_PRIVATE_REGISTRATION_CANCEL: null,
-	PURCHASES_PRIVATE_REGISTRATION_CANCEL_COMPLETED: null,
-	PURCHASES_PRIVATE_REGISTRATION_CANCEL_FAILED: null,
+	PRIVACY_PROTECTION_CANCEL: null,
+	PRIVACY_PROTECTION_CANCEL_COMPLETED: null,
+	PRIVACY_PROTECTION_CANCEL_FAILED: null,
+	PRIVACY_PROTECTION_ENABLE_COMPLETED: null,
```

Comment:
There are in fact other inconsistencies. For example the delete DNS action is plain wrong: [we're currently missing a `DNS_DELETE_COMPLETED` event](https://github.com/Automattic/wp-calypso/blob/de5df67d8c0f27782e9e379af6f86f8bfc30c205/client/lib/upgrades/actions/domain-management.js#L230) and [we're expecting an `error` object in the payload of the `DELETING_DNS` event](https://github.com/Automattic/wp-calypso/blob/de5df67d8c0f27782e9e379af6f86f8bfc30c205/client/lib/domains/dns/reducer.js#L48) that we never pass. And I'm not talking of how we handle errors :).

@fabianapsimoes, we should probably take care of that at some point.